.gitignore

```diff
@@ -3,7 +3,6 @@
 .bundle
 .config
 .yardoc
-Gemfile.lock
```

Comment:
I don't think we should push `Gemfile.lock` in the repo (see [this](https://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/))

Reply:
Interesting, I did that to prevent errors on Travis, which led me to think that I had to commit it. Hmmm.